governance/votes/Trial Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,191 @@
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Stability Scope Defined Parameter Changes
+
+As per the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104), and the [adjustments to the base rate](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6), the following parameter changes will be made, if this executive proposal passes:
+
+**[Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Decrease**
+- Decrease the DSR by 0.30% from 3.49% to 3.19%
+
+**[Liquidation Ratio](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio) Changes**
+- Reduce WSTETH-A Liquidation Ratio by 10% from 160% to 150%
+- Reduce WSTETH-B Liquidation Ratio by 10% from 185% to 175%
+- Reduce RETH-A Liquidation Ratio by 20% from 170% to 150%
+
+**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Reductions**
+- Reduce the ETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the ETH-B Stability Fee (SF) by 0.30% from 4.24% to 3.94%
+- Reduce the ETH-C Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the WSTETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WSTETH-B Stability Fee (SF) by 0.30% from 3.49% to 3.19%
+- Reduce the RETH-A Stability Fee (SF) by 0.30% from 3.74% to 3.44%
+- Reduce the WBTC-A Stability Fee (SF) by 0.11% from 5.80% to 5.69%
+- Reduce the WBTC-B Stability Fee (SF) by 0.11% from 6.30% to 6.19%
+- Reduce the WBTC-C Stability Fee (SF) by 0.11% from 5.55% to 5.44%
+
+#### CRVV1ETHSTETH-A 1st Stage Offboarding
+
+As per [MIP104 ](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope) the debt ceiling (``line``) for ``CRVV1ETHSTETH-A`` will be set to ``0``. 
+
+This change will be happening alongside the other [Stability Scope Parameter changes](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238/6).
+
+
+### Deploy Multiswap Conduit for RWA015-A (Andromeda)
+
+Following the results of a recent [technical assessment from ChainSecurity](https://github.com/makerdao/rwa-toolkit/blob/8d30ed2cb657641253d45b57c894613e26b4ae1b/audits/ChainSecurity_MakerDAO_RWA_Toolkit_audit.pdf), the following changes will be made to [BlackTower Andromeda RWA015](https://github.com/makerdao/rwa-toolkit) if this proposal passes:
+
+- A new version of the output conduit will be added to the chainlog as `RWA015_A_OUTPUT_CONDUIT`. This will allow an authorized party to choose which PSM they want to interact with.
```

Comment:
This can trigger mistaken assumptions as written. The implication here is that the act of adding it to the chainlog has resulted in something happening. This is not the case. The chainlog is more or less just a list of MakerDAO related contracts, it doesn't do anything by itself. Authorization happens directly on the relevant contracts, not via the chainlog. 

See the official version for more context on what is going on here.